CRM/Campaign/Selector/Search.php

```diff
@@ -282,7 +282,7 @@ public function buildPrevNextCache($sort) {
       );
       list($select, $from) = explode(' FROM ', $sql);
       $selectSQL = "
-      SELECT 'civicrm_contact', contact_a.id, contact_a.id, '$cacheKey', contact_a.display_name
+      SELECT '$cacheKey', contact_a.id, contact_a.display_name
```

Comment:
So this is compiled with
```
 $cacheKey = "civicrm search {$this->_key}";
```

I couldn't be certain from my checks so far that it's impossible to pass in the value via the url (it is set earlier in the change) - perhaps we should escape with alphanumeric to be sure - still this is not a new thing - it was already being passed like that

Reply:
Yeah, this is a good catch. I filed a separate issue for it.